jointstate_topic: /manipulator/joint_states

motor_topics:
  base_cyl: /roboclaw_driver/128/motor1/joint_state
  cyl_arm1: /roboclaw_driver/129/motor1/joint_state
  arm1_arm2: /roboclaw_driver/129/motor2/joint_state
  arm2_arm3: /roboclaw_driver/130/motor1/joint_state
  arm3_tool: /roboclaw_driver/130/motor2/joint_state

# home_offsets:
#   base_cyl: -0.35
#   cyl_arm1: -0.15
#   arm1_arm2: 1.5
#   arm2_arm3: -0.25
#   arm3_tool: -0.2

home_offsets:
  base_cyl: 0.0
  cyl_arm1: 0.0
  arm1_arm2: 0.0
  arm2_arm3: 0.0
  arm3_tool: 0.0

publish_rate: 20.0